lib/gql_serializer.rb

```diff
@@ -46,55 +46,71 @@ def self.query_include(model, hasharray)
     include_array
   end
 
-  def self.serialize(record, hasharray, options)
+  def self.serialize(record, hasharray, options, instructions = {})
 
     if record.nil?
       return nil
     end
 
+
     if record.respond_to? :map
       return record.map do |r|
-        self.serialize(r, hasharray, options)
+        self.serialize(r, hasharray, options, instructions)
       end
     end
 
-    hash = {}
-    model = record.class
-    all_relations = model.reflections.keys
+    id = "#{record.class}, #{hasharray}"
```

Comment:
I'm not sure about having a comma separator in an identifier like this, can you explain your thoughts here?

Reply:
With only two elements, the separator can be anything that wouldn't be found in the first element. The first element is an ActiveRecord class name that would not contain a comma. I'm open to other suggestions.

Reply:
When I made this comment, I was thinking of the comma and the space together and was thinking of such examples like Ruby object instances where they use symbols like `#`
```ruby
irb(main):002:0> ActiveRecord::Associations::Preloader.new
=> #<ActiveRecord::Associations::Preloader:0x00007fa2fc966d00 @associate_by_default=true>
```
But after speaking with you last week about this, I don't think this decision matters too much at the end of the day, fine  to leave it as is.